src/chrome/content/rules/Zillowstatic.com.xml

```diff
@@ -0,0 +1,11 @@
+<ruleset name="Zillowstatic.com">
+	<target host="zillowstatic.com" />
+	<target host="www.zillowstatic.com" />
+	<target host="photos1.zillowstatic.com" />
+	<target host="photos2.zillowstatic.com" />
+	<target host="photos3.zillowstatic.com" />
```

Comment:
Please add `wp.zillowstatic.com` with test url http://wp.zillowstatic.com/1/vert-one-resized-169851.jpg

Reply:
Oh, you actually added it but to the wrong file